src/app/clusters/identify-server/identify-server.cpp

```diff
@@ -183,13 +183,14 @@ static void onIdentifyClusterTick(chip::System::Layer * systemLayer, void * appS
         if (EMBER_ZCL_STATUS_SUCCESS == Clusters::Identify::Attributes::GetIdentifyTime(endpoint, &identifyTime) &&
             0 != identifyTime)
         {
+            identifyTime = static_cast<uint16_t>(identifyTime == 0 ? 0 : identifyTime - 1);
             // This tick writes the new attribute, which will trigger the Attribute
             // Changed callback below, which in turn will schedule or cancel the tick.
             // Because of this, the tick does not have to be scheduled here.
-            (void) Clusters::Identify::Attributes::SetIdentifyTime(endpoint,
-                                                                   static_cast<uint16_t>(identifyTime == 0 ? 0 : identifyTime - 1));
+            (void) Clusters::Identify::Attributes::SetIdentifyTime(endpoint, identifyTime);
         }
-        else
+
+        if (identifyTime == 0)
```

Comment:
Why is this not done with the Attributes Changed callback, as the comment above implies?

Reply:
Well, there are several ways of solving it. I just took the one, which would lead in less code changes.

Reply:
It is less code change but it conflicts with the comment.